apps/site/lib/site_web/controllers/page_controller.ex

```diff
@@ -13,6 +13,7 @@ defmodule SiteWeb.PageController do
     WhatsHappeningItem
   }
 
+  plug(:alerts)
```

Comment:
I haven't looked into it super closely, but was just curious if you considered including and/or modifying `plug(SiteWeb.Plugs.AlertsByTimeframe)`?

Reply:
🤔 I was aware of this, but didn't consider it too deeply. It looks like that plug expects an `alerts` assign on the `conn`, which is what this `:alerts` is doing currently, and then does some timeframe filtering which I don't think is applicable to this view? Dunno, if you have more thoughts on this let me know.